simple/stats/data.py

```diff
@@ -14,15 +14,22 @@
 
 from dataclasses import dataclass
 from dataclasses import field
+from urllib.parse import urlparse
 
 _PREDICATE_TYPE_OF = "typeOf"
 _PREDICATE_NAME = "name"
 _PREDICATE_DESCRIPTION = "description"
 _PREDICATE_MEMBER_OF = "memberOf"
 _PREDICATE_SPECIALIZATION_OF = "specializationOf"
+_PREDICATE_URL = "url"
+_PREDICATE_SOURCE = "source"
+_PREDICATE_DOMAIN = "domain"
+_PREDICATE_INCLUDED_IN = "includedIn"
```

Comment:
todo:  create a schema node for it in https://source.corp.google.com/piper///depot/google3/third_party/datacommons/schema/core/dcschema.mcf?q=f:dcschema.mcf?

Reply:
cl/582900903